com.avaloq.tools.ddk.xtext/src/com/avaloq/tools/ddk/xtext/resource/persistence/DirectLinkingResourceStorageLoadable.java

```diff
@@ -123,8 +125,6 @@ public void loadResource(final Resource resource) throws IOException {
       if (uri != null && uri.isHierarchical() && !uri.isRelative()) {
         baseURI = uri;
       }
-      boolean installDerivedState = ResourceSetOptions.installDerivedState(resourceSet); // must be read before readCompressedInt, see HACK comment inside

```

Comment:
do you know what the HACK was? Has it been removed by now?

Reply:
The HACK is still there, but it doesn't apply to setting installDerivedState any more